hack/update-helm.sh

```diff
@@ -17,24 +17,32 @@
 # Set the source and destination directories
 SRC_CRD_DIR=config/components/crd/bases
 SRC_RBAC_DIR=config/components/rbac
+SRC_WEBHOOK_DIR=config/components/webhook
 SRC_VISIBILITY_DIR=config/components/visibility
 
 DEST_CRD_DIR=charts/kueue/templates/crd
 DEST_RBAC_DIR=charts/kueue/templates/rbac
+DEST_WEBHOOK_DIR=charts/kueue/templates/webhook
 DEST_VISIBILITY_DIR=charts/kueue/templates/visibility
 
 YQ=./bin/yq
 
 # Create the destination directory if it doesn't exist
-mkdir -p ${DEST_CRD_DIR} ${DEST_RBAC_DIR} ${DEST_VISIBILITY_DIR}
+mkdir -p ${DEST_CRD_DIR} ${DEST_RBAC_DIR} ${DEST_WEBHOOK_DIR} ${DEST_VISIBILITY_DIR}
 
 # Add more excluded files separated by spaces
-EXCLUDE_FILES='kustomization.yaml'
+EXCLUDE_FILES='kustomization.yaml kustomizeconfig.yaml'
 
 # Copy all YAML files from the source directory to the destination directory
 cp ${SRC_CRD_DIR}/*.yaml ${DEST_CRD_DIR}
-find $SRC_VISIBILITY_DIR -name "*.yaml" $(printf "! -name %s " $EXCLUDE_FILES) -exec cp "{}" $DEST_VISIBILITY_DIR \;
 find $SRC_RBAC_DIR -name "*.yaml" $(printf "! -name %s " $EXCLUDE_FILES) -exec cp "{}" $DEST_RBAC_DIR \;
+find $SRC_WEBHOOK_DIR -name "*.yaml" $(printf "! -name %s " $EXCLUDE_FILES) -exec cp "{}" $DEST_WEBHOOK_DIR \;
+find $SRC_VISIBILITY_DIR -name "*.yaml" $(printf "! -name %s " $EXCLUDE_FILES) -exec cp "{}" $DEST_VISIBILITY_DIR \;
+$YQ -N -s '.kind' ${DEST_WEBHOOK_DIR}/manifests.yaml
```

Comment:
I don't think  `${DEST_WEBHOOK_DIR}/manifests.yaml` is used from this point on , remove it otherwise it remains untracked. 

Alternatively skip its copy and do the split form   `${SRC_WEBHOOK_DIR}/manifests.yaml`.

Reply:
Agree. I'll remove it.